pyuvdata/utils.py

```diff
@@ -3975,6 +4013,195 @@ def check_lsts_against_times(
         )
 
 
+def uvw_track_generator(
+    lon_coord=None,
```

Comment:
Let's go ahead and require all of these to be passed by name by inserting a "*," before the first parameter.

Reply:
Added!